jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/Updater.kt

```diff
@@ -48,7 +49,7 @@ class Updater(
     private var predicate: (StackResource) -> Boolean = { true }
     private val updating = AtomicBoolean(false)
     val running get() = updating.get()
-    private val eventsFetcher = EventsFetcher(stackName)
+    private val eventsFetcher = EventsFetcher(stackId)
```

Comment:
Why did we move from stackName to stackId?

Reply:
The DescribeStack and DescribeStackResources calls require the unique ID when called on deleted stacks. Since, ID can be used for regular calls as well, ID may be a better option than changing the parameters based on the status of the stacks